assets/build/install.sh

```diff
@@ -134,7 +153,7 @@ exec_as_git yarn install --production --pure-lockfile
 
 echo "Compiling assets. Please be patient, this could take a while..."
 #Adding webpack compile needed since 8.17
-exec_as_git bundle exec rake assets:clean assets:precompile webpack:compile USE_DB=false SKIP_STORAGE_VALIDATION=true >/dev/null 2>&1
+exec_as_git bundle exec rake assets:clean assets:precompile webpack:compile USE_DB=false SKIP_STORAGE_VALIDATION=true
```

Comment:
I'm not sure I understand. Do you mean this?
```
diff --git a/assets/runtime/functions b/assets/runtime/functions
index 7d4a647..d372861 100644
--- a/assets/runtime/functions
+++ b/assets/runtime/functions
@@ -1525,7 +1525,7 @@ migrate_database() {
     # assets need to be recompiled when GITLAB_RELATIVE_URL_ROOT is used
     if [[ -n ${GITLAB_RELATIVE_URL_ROOT} ]]; then
       echo "Recompiling assets (relative_url in use), this could take a while..."
-      exec_as_git bundle exec rake assets:clean assets:precompile webpack:compile  >/dev/null 2>&1
+      exec_as_git bundle exec rake gitlab:assets:compile USE_DB=false SKIP_STORAGE_VALIDATION=true
     fi

     echo "Clearing cache..."
```
Meaning that `gitlab:assets:compile` equals `assets:precompile webpack:compile`?
@solidnerd